test/extended/testdata/builds/test-build.yaml

```diff
@@ -223,19 +223,3 @@ items:
     nodeSelector: 
   status:
     lastVersion: 0
-- kind: BuildConfig
```

Comment:
this was only used previously by cluster-config.go

but other BC's here were used by other tests